deployment/README.md

```diff
@@ -130,7 +176,7 @@ Extend your `docker-compose.yaml` with the following service:
     build:
       context: ./source
     volumes:
-      - ./data/api-data:/data
+      - ./data/api-data:/opt/obs/data
```

Comment:
I have not needed this in my migrated production deployment - are you sure? The other container also expects the api-data to be in ``/data``. I was able to parse track files from ``/data``.